xsct.c

```diff
@@ -218,10 +219,12 @@ int main(int argc, char **argv)
         }
         else
         {
-            temp = (temp == 0) ? TEMPERATURE_NORM : temp;
-
-            for (screen = 0; screen < screens; screen++)
+            temp = (temp == 0) ? TEMPERATURE_NORM : temp; 
+            for (screen = 0; screen < screens; screen++) 
+            {
+                temp = (temp > 1000) ? temp : (temp + (temp != 0)*get_sct_for_screen(dpy, screen, fdebug)); 
```

Comment:
```suggestion
            for (screen = 0; screen < screens; screen++) 
            {
                temp = (temp < 1000) ? ((temp == 0) ? TEMPERATURE_NORM : temp + get_sct_for_screen(dpy, screen, fdebug)) : temp; 
```
that's harder to read with the nested ternary operator

Reply:
:question: What for?
```c
            temp = (temp == 0) ? TEMPERATURE_NORM : temp; 
            for (screen = 0; screen < screens; screen++) 
            {
                temp = (temp > 1000) ? temp : (temp + get_sct_for_screen(dpy, screen, fdebug)); 
```
See https://github.com/faf0/sct/pull/14#discussion_r438539256

Reply:
> What for?

Readability.

The suggested code potentially self-assigns `temp` to `temp` once with each ternary operator. An if-then-else cascade makes the cases clear and doesn't re-assign unnecessarily.

Agree that an extra command-line flag might be a better solution.